webviews/src/BranchBasePicker.tsx

```
@@ -0,0 +1,528 @@
+        ) : (
+          <BranchBaseButtonLabel label={label} picker={picker} />
+        )}
+      </button>
+      {open ? (
+        /* oxlint-disable-next-line jsx-a11y/prefer-tag-over-role */
+        <div className="base-picker-popover" role="dialog" aria-label={label("branchPickerOpen")}>
+          <div className="base-picker-search">
+            <Icon name="search" />
+            <input
+              ref={inputRef}
+              autoFocus
```

Biome (2.5.0)

[error] Avoid the autoFocus attribute.

(lint/a11y/noAutofocus)